frontend/src/components/TransactionLinks/TransactionLink.vue

```diff
@@ -22,6 +22,10 @@
                 <b-icon icon="clipboard"></b-icon>
                 {{ $t('gdd_per_link.copy') }}
               </b-dropdown-item>
+              <b-dropdown-item v-if="validLink" class="test-copy-link" @click="copyLinkWithText()">
```

Comment:
```suggestion
              <b-dropdown-item v-if="validLink" class="test-copy-text pt-3" @click="copyLinkWithText()">
```

Reply:
this is used in the tests to find the click

Reply:
**pt-3** ist der **padding-top** von bootstrap

das die abstände passen 

![image](https://user-images.githubusercontent.com/1324583/177257979-caba100b-be63-449c-ade0-8e9ad0bf6318.png)